app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(17, 4)) == 17, \
+        "sum of returned list should be equal value"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert split_integer(30, 6) == [5, 5, 5, 5, 5, 5], \
+        "value should be split on equal parts"
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert split_integer(8, 1) == [8], \
+        "when number of parts equal 1 in the list should be only one part"
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    assert split_integer(32, 6) == [5, 5, 5, 5, 6, 6], \
+        "parts of number should be sorted from min to max"
 
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+    assert split_integer(1, 5) == [0, 0, 0, 0, 1], \
+        "when value is less than number of parts in the list should be zero"
```

Comment:
You are creating an unnecessary variable in your assert statement. You should directly compare the result of the function call to the expected result. Refer to the Code Style section of the checklist.